install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -3018,8 +3018,13 @@ authentication:
               country: "US"
               organization: "SPIRE"
               commonName: "Cilium SPIRE CA"
-      # -- SPIRE server address
-      serverAddress: spire-server.cilium-spire.svc:8081
+      # -- SPIRE server address used by Cilium Operator
+      #
+      # As Cilium Operator is running in host network, please adjust operator.dnsPolicy to
+      # ClusterFirstWithHostNet if DNS resolution is required.
```

Comment:
This needs expansion, since the operator now looks up service names.

Reply:
It seems like this description is causing confusion now, removing the mention of operator.dnsPolicy.